e2e-test/functions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+PID=0
+function finish() {
+  if [[ ${PID} != 0 ]]; then
+    kill ${PID} || true
```

Comment:
Probably not required here; `set -e` does not descend. Also, use `|| :` instead of `|| true` since `:` is a builtin noop (results are the same)

Reply:
I'm not sure what you mean. Since I'm starting the operator in the background, I want to make sure it is being stopped even after encountering an error.